rpc/jsonrpc/client/http_json_client.go

```diff
@@ -174,28 +180,23 @@ func NewWithHTTPClient(remote string, c *http.Client) (*Client, error) {
 
 // Call issues a POST HTTP request. Requests are JSON encoded. Content-Type:
 // application/json.
-func (c *Client) Call(
-	ctx context.Context,
-	method string,
-	params map[string]interface{},
-	result interface{},
-) (interface{}, error) {
+func (c *Client) Call(ctx context.Context, method string, params, result interface{}) error {
```

Comment:
I kind of wish we handled nil params, rather than forcing a lot of the call sites to construct empty maps for kicks.

Reply:
Yeah. Actually I think we could correctly do so right now, but I didn't want to start messing with that in this change. That _will_ work once I'm further along.